treeherder/etl/th_publisher.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from django.utils.encoding import python_2_unicode_compatible
+from django.conf import settings
+
+from thclient import TreeherderRequest
+
+from treeherder.etl.oauth_utils import OAuthCredentials
+
+
+logger = logging.getLogger(__name__)
+
+
+def post_treeherder_collections(th_collections):
+    errors = []
+    for project in th_collections:
+
+        credentials = OAuthCredentials.get_credentials(project)
+
+        th_request = TreeherderRequest(
+            protocol=settings.TREEHERDER_REQUEST_PROTOCOL,
+            host=settings.TREEHERDER_REQUEST_HOST,
+            project=project,
+            oauth_key=credentials.get('consumer_key', None),
+            oauth_secret=credentials.get('consumer_secret', None)
+        )
+
+        logger.info(
+            "collection loading request: {0}".format(
+                th_request.get_uri(th_collections[project].endpoint_base)))
+        response = th_request.post(th_collections[project])
+
+        if not response or response.status not in [200, "200 OK"]:
```

Comment:
Just curious here, why are we expecting the "200 OK" string as a response status?